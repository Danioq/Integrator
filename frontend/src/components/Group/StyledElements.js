import styled from 'styled-components';

export let GroupContainer = styled.ul`
  list-style: none;
  `;

export let GroupHeader = styled.p`
  font-size: 1.5em;
  font-weight: bold;
  cursor: default;

`;
