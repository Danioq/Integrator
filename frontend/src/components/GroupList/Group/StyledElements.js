import styled from 'styled-components';

export let GroupContainer = styled.div``;