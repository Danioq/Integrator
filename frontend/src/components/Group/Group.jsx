import React from 'react';
import Employee from 'components/Employee'
import {GroupContainer, GroupHeader} from './StyledElements'


export default function Group(props){
  return <React.Fragment>
    <GroupContainer>
      <GroupHeader>Grupa {props.number}</GroupHeader>
      {props.employees.map((employeeData, index) => <Employee key={index} employee={employeeData}/>)}
    </GroupContainer>
  </React.Fragment>;
}