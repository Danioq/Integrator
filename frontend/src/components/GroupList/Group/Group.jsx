import React from 'react';
import Employee from './Employee'
import {GroupContainer} from './StyledElements'


export default function Group(props){
  return <GroupContainer>
    {props.employees.map(employeeData => <Employee employee={employeeData}/>)}
  </GroupContainer>;
    ;
}