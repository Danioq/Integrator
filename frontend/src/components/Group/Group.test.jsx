import Group from './Group';
import renderer from 'react-test-renderer';
import React from 'react';

describe('Group component suite', () => {
  test('Render test', () => {
    let component = renderer.create(<Group employees={[]} number={1}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});