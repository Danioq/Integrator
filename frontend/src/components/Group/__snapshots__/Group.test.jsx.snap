// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group component suite Render test 1`] = `
<ul
  className="sc-AxheI kDXpPc"
>
  <p
    className="sc-Axmtr gpYXoQ"
  >
    Grupa 
    1
  </p>
</ul>
`;
